clinica/converters/aibl_to_bids/utils/json.py

```diff
@@ -0,0 +1,340 @@
+            [
+                "SpecificRadioactivity",
+                (
+                    "RadiopharmaceuticalInformationSequence",
+                    "RadiopharmaceuticalSpecificActivity",
+                ),
+                "n/a",
+            ],
+            ["SpecificRadioactivityUnits", (), "Bq/micromole"],
+            [
+                "ModeOfAdministration",
+                (),
+                "n/a",
+            ],
+            # Reconstruction
+            [
+                "AcquisitionMode",
+                (),
+                "n/a",
+            ],  # todo : could not find / (0054, 1000) Series Type ?
+            ["ImageDecayCorrected", ("DecayCorrection",), "n/a"],
+            ["ImageDecayCorrectionTime", ("DecayCorrection",), np.nan],
+            ["ReconMethodName", ("ReconstructionMethod",), "n/a"],
+            [
+                "ReconMethodParameterLabels",
+                ("ReconstructionMethod",),
+                [""],
+            ],  # todo : could not find
+            ["ReconMethodParameterUnits", ("ReconstructionMethod",), [""]],
+            ["ReconMethodParameterValues", ("ReconstructionMethod",), [""]],
+            ["ReconFilterType", ("ConvolutionKernel",), "n/a"],
+            [
+                "ReconFilterSize",
+                ("ConvolutionKernel",),
+                np.nan,
+            ],  # todo : could not find
+            ["AttenuationCorrection", ("AttenuationCorrectionMethod",), "n/a"],
+        ],
+    ).set_index(keys="BIDSname", drop=False)
+
+
+def _get_dicom_tags_and_defaults_base() -> pd.DataFrame:
+    return pd.DataFrame(
+        columns=["BIDSname", "DCMtag", "Value"],
+        data=[
+            # Hardware
+            ["DeviceSerialNumber", ("DeviceSerialNumber",), "n/a"],
+            ["ManufacturersModelName", ("ManufacturerModelName",), "n/a"],
+            ["SoftwareVersions", ("SoftwareVersions",), "n/a"],
+            ["BodyPart", ("BodyPartExamined",), "n/a"],
+            ["MagneticFieldStrength", ("MagneticFieldStrength",), "n/a"],
+            # Institution
+            ["InstitutionName", ("InstitutionName",), "n/a"],
+            ["InstitutionAddress", ("InstitutionAddress",), "n/a"],
+            [
+                "InstitutionalDepartmentName",
+                ("InstitutionalDepartmentName",),
+                "n/a",
+            ],
+        ],
+    ).set_index(keys="BIDSname", drop=False)
+
+
+def _check_dcm_value(
+    dicom_data: Optional[Any],
+) -> Union[None, str, list, float]:
+    """Handles case where result is MultiValue, which is not JSON serializable and would be replaced by a blank space"""
+    if dicom_data:
+        if type(dicom_data) == MultiValue:
+            return list(dicom_data)
+        return dicom_data
+    return None
+
+
+def _fetch_dcm_data_from_header(
+    dicom_header: Union[FileDataset, DataElement], dicom_tags: Optional[tuple[str, ...]]
+) -> Optional[Union[Any, DataElement]]:
```

Comment:
Here, however, the union of `Any` with something is always `Any`, so let's go with:

```suggestion
) -> Optional[Any]:
```